flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  description = "Dependencies for development";
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs, flake-utils } @ imports: (
+    flake-utils.lib.eachDefaultSystem (
+      system: (
+        let
+          pkgs = import nixpkgs { inherit system; };
+          baseDependencies = with pkgs; [
+            python3
```

Comment:
We need python? :O

Reply:
For the Git hooks. I was also a little surprised but it just didn't work when I tried to commit on my laptop

Reply:
I don't get what's using python there, but makes sense to have it then